conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -18,3 +18,5 @@ GET   /docs/swagger-ui/*file                controllers.Assets.at(path:String="/
 GET   /assets/*file                         controllers.Assets.versioned(path="/public", file: Asset)
 
 ->         /                                prod.Routes
+
+DELETE    /delete/conversation/:conversationId/:client    uk.gov.hmrc.securemessage.controllers.TestOnlyController.deleteConversation(conversationId, client)
```

Comment:
referencing the same package across multiple route files causes issues with the route generator.